README.md

```diff
@@ -9,3 +9,20 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+
+///
+
+Assuming that the number of nodes in graph A and graph B is the same, n, and there must be an edge between every two nodes, then the structure of their edges will be consistent no matter how they are arranged.
+
+|VA| = |VB| = n, then bijection f: VA -> VB 
+
+for any u,v ∈ VA, (u,v) ∈EA <-> (f(u),F(v)∈EB)
+
+f is a bijection and preserves the edge structure, so it is isomorphic as long as the number of nodes is the same.
```

Comment:
How do you know this? You haven't said anything about how to get this f, or how it has to be a bijection.